standard_test.go

```diff
@@ -898,6 +911,7 @@ func TestEliminateSnakes(t *testing.T) {
 				NotEliminated,
 				EliminatedByHeadToHeadCollision,
 			},
+			[]string{"2", "1", "", "1"},
```

Comment:
This one is weird and non-intuitive. You would expect them all to be eliminated by snake 3 but because of how resolution is handled (we just iterate through the list of snakes) all eliminations are with the first snake on that head square and not the one that might live.

Reply:
Interesting. I agree it should be all eliminated by 3. Think this is worth correcting while we're here? We'd have to update the code to look for the longest length head to head?

Reply:
Yeah I'll fix it to do that.